specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-10-01-preview/machineLearningServices.json

```diff
@@ -3719,6 +3719,11 @@
             "name"
           ]
         },
+        "osImageMetadata": {
+          "readOnly": true,
+          "description": "Returns the metadata about the current and latest version for this ComputeInstance",
```

Comment:
"Returns metadata about the operating system image for this compute instance." < Let's use lower casing for compute instance in alignment with marketing/cela naming requirements.